examples/build.gradle

```diff
@@ -1,16 +1,68 @@
 description = 'OpenCensus Examples'
 
+buildscript {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "gradle.plugin.com.github.sherter.google-java-format:google-java-format-gradle-plugin:0.6"
+    }
+}
+
+apply plugin: "checkstyle"
+apply plugin: 'idea'
+apply plugin: 'java'
+// Plugins that require java8
+if (JavaVersion.current().isJava8Compatible()) {
+    apply plugin: 'com.github.sherter.google-java-format'
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+group = "io.opencensus"
+version = "0.11.0-SNAPSHOT"
+
+def opencensusVersion = "0.10.1" // LATEST_OPENCENSUS_RELEASE_VERSION
+
+checkstyle {
+    configFile = file("$rootDir/../buildscripts/checkstyle.xml")
+    toolVersion = "8.0"
+    ignoreFailures = false
+    if (rootProject.hasProperty("checkstyle.ignoreFailures")) {
+        ignoreFailures = rootProject.properties["checkstyle.ignoreFailures"].toBoolean()
+    }
+    configProperties["rootDir"] = file("$rootDir/../")
+}
+
+// Disable checkstyle if no java8.
+checkstyleMain.enabled = JavaVersion.current().isJava8Compatible()
+checkstyleTest.enabled = JavaVersion.current().isJava8Compatible()
```

Comment:
Should we assume that the Java version is at least 8 here, since the source also requires Java 8?

Reply:
Done.